src/actions/channels.js

```diff
@@ -467,6 +467,35 @@ export function getChannelAndMyMember(channelId) {
     };
 }
 
+export function getChannelTimezones(channelId) {
+    return async (dispatch, getState) => {
+        dispatch({type: ChannelTypes.GET_CHANNELS_TIMEZONE_REQUEST}, getState);
+
+        let channelTimezones;
+        try {
+            const channelTimezonesRequest = Client4.getChannelTimezones(channelId);
+
+            channelTimezones = await channelTimezonesRequest;
+        } catch (error) {
+            forceLogoutIfNecessary(error, dispatch, getState);
+            dispatch(batchActions([
+                {type: ChannelTypes.GET_CHANNELS_TIMEZONE_FAILURE, error},
+                logError(error)(dispatch),
```

Comment:
Similarly, we don't need to explicitly pass `dispatch` into this since it'll be handled by the outer `dispatch`

Reply:
done